app/forms.py

```diff
@@ -0,0 +1,11 @@
+from flask_wtf import FlaskForm
+from wtforms import StringField, IntegerField, SubmitField, DateTimeLocalField
+from wtforms.validators import DataRequired
+
+class ChallengeCreationForm(FlaskForm):
+    title = StringField('Challenge Title', validators=[DataRequired()])
+    start = StringField('Starting Article', validators=[DataRequired()])
+    destination = StringField('Destination Article', validators=[DataRequired()])
+    time = DateTimeLocalField('Submission Close', validators=[DataRequired()])
+    attempts = IntegerField('Submission Attempts', validators=[DataRequired()])
```

Comment:
Is this field necessary (attempts) ? I dont think it is doing anything

Reply:
Also, please do a local merge, as this forms.py already should exist from an earlier PR